Common/Cloud-Account/aws-cfn-cloud-account-connector/cloudone.template.yaml

```diff
@@ -51,63 +61,61 @@ Resources:
           CloudOneApiKey: !Ref CloudOneApiKey
       Code:
         ZipFile:
-          !Sub
-            |-
-              import json
-              import os
-              import urllib3
-              import boto3
-              import cfnresponse
-
-              def lambda_handler(event, context):
-                  status = cfnresponse.SUCCESS
-                  response_data = {}
-                  physicalResourceId = None
-                  try:
-
-                      cloudOneRoleArn = os.environ['CloudOneRoleArn']
-                      cloudOneRegion = os.environ['CloudOneRegion']
-                      cloudOneApiKey = os.environ['CloudOneApiKey']
-
-                      headers = {
-                        'api-version': 'v1',
-                        'Authorization': 'ApiKey '+cloudOneApiKey+'',
-                        'Content-Type': 'application/json'
-                      }
+          |-
+            import json
+            import os
+            import urllib3
+            import cfnresponse
+
+            def lambda_handler(event, context):
+                status = cfnresponse.SUCCESS
+                response_data = {}
+                physicalResourceId = None
+                try:
+
+                    cloudOneRoleArn = os.environ['CloudOneRoleArn']
+                    cloudOneRegion = os.environ['CloudOneRegion']
+                    cloudOneApiKey = os.environ['CloudOneApiKey']
```

Comment:
Was looking through the templates and maybe I missed it:
Is this API Key being encrypted?
I can only see it being flagged as NoEcho in the params value adds